src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -1646,6 +1652,14 @@ lambdaBody
     |    statement)
     ;
 
+textBlock
+    :   !c:TEXT_BLOCK_CONTENT
```

Comment:
I'm not sure why the "TEXT_BLOCK_LITERAL_BEGIN" is not returned first before the token stream gets pushed to the text block lexer, since this is the intended behavior (see [this](https://www.antlr2.org/doc/streams.html)).  What I've done here works fine, but is not ideal.